packages/global/browser/braze-preference-center.vue

```diff
@@ -167,6 +195,17 @@ export default {
       if (this.timeout) clearTimeout(this.timeout);
       this.timeout = setTimeout(this.setOptIns, 300);
     },
+    async loadRecaptcha() {
+      try {
+        this.recaptcha.loading = true;
+        this.recaptcha.error = null;
+        await recaptchaLoad({ siteKey: this.siteKey });
+      } catch (e) {
+        this.recaptcha.error = e;
+      } finally {
+        this.recaptcha.loading = false;
+      }
+    },
```

Comment:
No need for separate and error/loading states here. 
```suggestion
    async loadRecaptcha() {
      try {
        this.loading = true;
        await recaptchaLoad({ siteKey: this.siteKey });
      } catch (e) {
        this.error = e.message;
      } finally {
        this.loading = false;
      }
    },
```